cli/src/movement_util.rs

```diff
@@ -179,12 +179,17 @@ pub fn move_to_commit(
     let current_wc_id = workspace_command
         .get_wc_commit_id()
         .ok_or_else(|| user_error("This command requires a working copy"))?;
-    let edit = edit
-        || !&workspace_command
-            .repo()
-            .view()
-            .heads()
-            .contains(current_wc_id);
+    let edit = match ui.movement_edit_mode() {
+        MovementEditMode::Always => true,
+        MovementEditMode::Never => false,
+        MovementEditMode::Auto => {
+            edit || !&workspace_command
+                .repo()
+                .view()
+                .heads()
+                .contains(current_wc_id)
+        }
+    };
```

Comment:
Looks like `--edit` is now ignored. I think we should either remove the flag or let the flag override the config (and provide `--no-edit` to override in the other direction)

Reply:
Or just respect the flag in all cases?

Reply:
Yes, but how is that different from what I said?

Reply:
My bad I only saw the last part of the message.

Reply:
Fixed properly now. Thanks.